lib/Clam/CfgBuilder.cc

```diff
@@ -1885,6 +1894,33 @@ void CrabIntraBlockBuilder::visitSelectInst(SelectInst &I) {
   }
 }
 
+/* alloc site fat ptr construction */
```

Comment:
It would be nice to extend the comment. 
Given an instruction `p := alloc(sz)` then the following CrabIR is added:
```
translate(p:= alloc(sz));
p_offset := 0;
p_size := translate(sz);
```

where `translate(code)` is the current translation from LLVM `code` to CrabIR